manifests/opendatahub/overlays/odh-model-controller/manager/kustomization.yaml

```diff
@@ -8,13 +8,10 @@ configMapGenerator:
 - files:
   - controller_manager_config.yaml
   name: manager-config
-- files:
-  - servingruntimes_config.yaml
-  name: servingruntimes-config
 
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
   newName: quay.io/opendatahub/odh-model-controller
-  newTag: v0.9.5-auth
+  newTag: v0.9.3-auth
```

Comment:
Is the tag downgrading from .5 to .3? In upstream (ODH) repo it is ` newTag: v0.9.5-auth`

Reply:
Hey, yep, I'm just getting both repos in sync. This tag doesn't actually matter as it uses a variable to select what image is deployed.